core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -319,15 +319,15 @@ private[spark] class AppStatusListener(
     }
 
     val lastStageInfo = event.stageInfos.sortBy(_.stageId).lastOption
-    val lastStageName = lastStageInfo.map(_.name).getOrElse("(Unknown Stage Name)")
+    val jobName = lastStageInfo.map(_.name).getOrElse("(Unknown Job without stages)")
```

Comment:
Nit: Sorry, but I think we can change the `Job` to lowercase, or change the `stages` to uppercase. It is my fault on the text-transform.